### DISCUSSION TITLE
Calling peaks per cell-type (cluster) joint multiome analysis

### DISCUSSION THREAD

**Discussion by drosop** [stuart-lab/signac#1430] [Q&A] [answered]
Hi, 

I have 10x multiome data (snATAC, snRNA) following this tutorial (https://stuartlab.org/signac/articles/pbmc_multiomic.html).

After getting WNN clusters, I want to perform downstream snATAC analysis including DARs and chromVAR analysis. 

Should I call peaks again using macs2 and group.by = "cell type"? or intial peaks called per sample using macs2 should be fine for downstream analysis? 

I don't see any strong batch effect in my data.

Thank you,

**Comment by timoast** [comment, marked as answer]
Generally we find that running cell-type-specific peak calling can identify some peaks for more rare cell types that may be missed when calling peaks on the bulk population. There's some analysis of this in the Signac paper: https://www.nature.com/articles/s41592-021-01282-5
